### PR TITLE
Render  optimization

### PR DESCRIPTION
**Pull request by gitesthuman**
# Pull request review checklist 📝 ✅

This is a checklist that helps conduct a pull request review properly. These points are required to merge the PR

- [x] There are no merge conflicts

- [x] The pull request is named accordingly (description of the functionality - it
   will be visible in the changelog later)
   
- [x] Every task requirement (from Jira) is done

- [ ] The app runs, it boots up properly (e.g. no syntax errors)

- [ ] The proposed functionality is working properly (you need to pull this branch
   and check for yourself)

- [ ] Each function, class or functionality is properly documented
   (short description of what it does, parameters or return values if they need
   any explanation)

### CODE REVIEW

File: dnd-bot/dnd_bot/logic/prototype/multiverse.py [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import copy
+import numpy as np
 
 
 class Multiverse:
     """contains the list of all the games (objects of Game class) that are being played"""
     games = dict()
+    square_size = 50
+    view_range = 4
```
Comment (Silentsky0) [inline dnd-bot/dnd_bot/logic/prototype/multiverse.py:9]: może te wartości wywalić jako stałe do jsona?

File: dnd-bot/dnd_bot/logic/prototype/multiverse.py [hunk 2]
```diff
@@ -12,3 +16,22 @@ def get_game(token):
     @staticmethod
     def add_game(game) -> None:
         Multiverse.games[game.token] = copy.deepcopy(game)
+
+    @staticmethod
+    def generate_masks():
```
Comment (Silentsky0) [inline dnd-bot/dnd_bot/logic/prototype/multiverse.py:21]: dokumentacja tej funkcji

File: dnd-bot/dnd_bot/logic/prototype/multiverse.py (hunk 1 above)
Comment (gitesthuman) [reply dnd-bot/dnd_bot/logic/prototype/multiverse.py:9]: wydaje mi się, że jest to niepotrzebne

Comment (gitesthuman) [reply dnd-bot/dnd_bot/logic/prototype/multiverse.py:9]: te wartości są stałe dla wszystkich gier

Comment (Silentsky0) [reply dnd-bot/dnd_bot/logic/prototype/multiverse.py:9]: no to stałe pythonowe poza funkcją

**Review by Silentsky0 — APPROVED** [review]

**Pull request merged**